### PR TITLE
Issue #2803257: OrderRefresh must not touch already completed orders

### PR DESCRIPTION
**Pull request by mglaman**

### CODE REVIEW

File: modules/order/src/OrderRefresh.php [hunk 1]
```diff
@@ -71,6 +71,12 @@ public function addProcessor(OrderProcessorInterface $processor, $priority) {
    * {@inheritdoc}
    */
   public function needsRefresh(OrderInterface $order) {
+
```
Comment (bojanz) [inline modules/order/src/OrderRefresh.php:74]: Extra newline? :P

File: modules/order/tests/src/Kernel/OrderRefreshTest.php [hunk 2]
```diff
@@ -193,6 +193,13 @@ public function testOrderCanRefresh() {
     $this->assertFalse($this->orderRefresh->needsRefresh($this->order));
     $this->order->setOwner($this->user);
     $this->assertTrue($this->orderRefresh->needsRefresh($this->order));
+
+    sleep(1);
+    $workflow = $this->order->getState()->getWorkflow();
+    $this->order->getState()->applyTransition($workflow->getTransition('place'));
+    $this->order->save();
+    $this->assertFalse($this->orderRefresh->needsRefresh($this->order));
+
```
Comment (bojanz) [inline modules/order/tests/src/Kernel/OrderRefreshTest.php:202]: And another one!

**Review by bojanz — CHANGES_REQUESTED** [review]

File: modules/order/src/OrderRefresh.php (hunk 1 above)
Comment (mglaman) [reply modules/order/src/OrderRefresh.php:74]: I'll get you.

**Comment by bojanz** [comment]
Committed in f2bf286979dda5986fc0b47ee77c96fe5dc480f3.

**Pull request closed without merging**